### PR TITLE
[FIX] web: paste text in input field with no line break

### PR DESCRIPTION
**Pull request by juliusc2066**
This commit fixes an issue that allowed users to paste text containing line breaks inside of a no line break input (like text fields with line_break = False option). This is resolved by replacing line breaks by white spaces on input when the event originates from a paste action.

Task 4130435

### CODE REVIEW

File: addons/web/static/tests/views/fields/text_field_tests.js [hunk 1]
```diff
@@ -675,4 +675,27 @@ QUnit.module("Fields", (hooks) => {
 
         assert.ok(afterHeight > initialHeight, "Should be taller than one character");
     });
+
+    QUnit.debug("text field without line breaks", async function (assert) {
```
Comment (aab-odoo) [inline addons/web/static/tests/views/fields/text_field_tests.js:679]: debug

**Review by aab-odoo — APPROVED** [review]
add task id in commit message

**Comment by aab-odoo** [comment]
robodoo delegate+

**Comment by juliusc2066** [comment]
@robodoo r+

**Pull request closed without merging**